### PR TITLE
add tag in all communities page

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: src/containers/TagInAllCommunities.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { prefetch } from 'react-fetcher'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import CoverImagePage from '../components/CoverImagePage'
+import TagPosts from './community/TagPosts'
```
Comment (levity) [inline src/containers/TagInAllCommunities.js]: TagPosts shouldn't be in the community subfolder anymore

File: src/containers/community/TagPosts.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ const TagPosts = props => {
   return <div>
     <div className='list-controls tag-header'>
       <span className='tag-name'>#{tagName}</span>
-      {tag.followed
+      {id && (tag.followed
         ? <button className='unfollow' onClick={() => dispatch(followTag(id, tagName))}>Unfollow</button>
-      : <button className='follow' onClick={() => dispatch(followTag(id, tagName))}>Follow</button>}
+      : <button className='follow' onClick={() => dispatch(followTag(id, tagName))}>Follow</button>)}
```
Comment (levity) [inline src/containers/community/TagPosts.js]: indentation's off here

**Comment by robbiecarlton** [comment]
@levity made those changes, good to merge?

**Comment by levity** [comment]
👌

**Pull request merged**